modules/top-level/nixpkgs.nix

```diff
@@ -165,10 +275,21 @@ in
       # evaluate the wrapper to find out that the priority is lower, and then we
       # don't need to evaluate `finalPkgs`.
       _module.args.pkgs = lib.mkOverride lib.modules.defaultOverridePriority finalPkgs.__splicedPackages;
-
-      # FIXME: This is a stub option for now
-      warnings = lib.optional (
-        opt.source.isDefined && opt.source.highestPrio < (lib.mkOptionDefault null).priority
-      ) "Defining the option `nixpkgs.source` currently has no effect";
     };
+
+  assertions = [
```

Comment:
does this need to go up a level into the {} above

Reply:
Yeah, my bad. Just pushed a fix as you commented :sweat_smile: